README.md

```diff
@@ -1,5 +1,22 @@
-# Morpho Blue IRM
+# Morpho Blue IRMs
+
+Some interest rate models for Morpho Blue:
```

Comment:
my formatter (Prettier) inserts a newline after this line

Reply:
Should we use prettier to format markdown files?